packages/pigeon/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 14.0.1
```

Comment:
Isn't this change breaking? I would expect that code that caught `FlutterError`s when calling Flutter APIs would compile but never actually catch errors after this change.